docs/guides/cloud/introduction.mdx

```diff
@@ -4,157 +4,152 @@ sidebar_position: 10
+integration with self-managed
+[GitHub](https://docs.github.com/en/enterprise-server@3.5/admin/overview/about-github-enterprise-server)
+and [GitLab](https://docs.gitlab.com/ee/subscriptions/self_managed/) instances
+too.
 
-We also integrate with two of the world's most popular collaboration tools:
-[Slack](https://slack.com/) and [Jira](https://www.atlassian.com/software/jira).
-Deliver test results with valuable additional context, directly into a dedicated
-Slack channel. Or for those on the
-[Team, Business or Enterprise plans](https://www.cypress.io/pricing/),
-[create bidirectionally linked Jira tickets](/guides/cloud/integrations/jira#Creating-a-Jira-issue-for-a-test-case)
-directly from specific test failures.
+#### Team Communication
+
+Get notified on test progress directly in your team's
+[Slack](/guides/cloud/integrations/slack) or
+[Microsoft Teams](/guides/cloud/integrations/teams) channels.
 
 <DocsImage
   src="/img/dashboard/slack-integration/cypress-slack-integration-channel-feed.jpg"
   alt="Cypress notification feed in Slack channel"
   width={560}
 />
 
-Lastly you can use our flexible adminstrative functions to configure Cypress
-Cloud however you want, grouping
+#### Issue Management
+
+Keep the whole team up-to-date with [Jira](/guides/cloud/integrations/jira)
+integration. You can
+[create bidirectionally linked Jira tickets](/guides/cloud/integrations/jira#Creating-a-Jira-issue-for-a-test-case)
+directly from specific test failures.
+
+### Cypress app integration
+
+The [Cypress app](/guides/cloud/recorded-runs#Cypress-App) integrates directly with
+Cypress Cloud to provide developers with the latest test results without having
+to switch tools.
+
+<DocsImage
+  src="/img/guides/core-concepts/cypress-app/debug-page.png"
+  alt="Debug"
+/>
+
+### Flexible enterprise configuration and single sign on
+
+You can use our flexible administrative functions to configure Cypress Cloud however you want, grouping
 [projects](/guides/cloud/account-management/projects) into multiple
 [organizations](/guides/cloud/account-management/organizations) if you have a
 lot, checking [usage](/guides/cloud/account-management/billing-and-usage), and
 administering
-[users and permissions](/guides/cloud/account-management/users#User-roles). We
-also provide [SSO integration](/guides/cloud/account-management/enterprise-sso)
+[users and permissions](/guides/cloud/account-management/users#User-roles).
+
+We also provide [SSO integration](/guides/cloud/account-management/enterprise-sso)
 for teams on our [Enterprise plan](https://www.cypress.io/pricing/).
 
+## Example project
+
+The Cypress
+[Real World App (RWA)](https://github.com/cypress-io/cypress-realworld-app)
+leverages [Cypress Cloud in CI](https://cloud.cypress.io/projects/7s5okt) to
+test over 300 test cases in parallel across 25 machines, multiple browsers,
+multiple device sizes and multiple operating systems.
+
+---
```

Comment:
I think we can kill this <hr>. We don't do this anywhere else and feels a little heavy. I think it flows fine without the break.